### PR TITLE
NCSDK-28025: suit: Switch to ipc_radio

### PR DESCRIPTION
**Pull request by tomchy**
Based on the conversation with @KAGA164 and @ppryga-nordic, everyone in NCS has been migrated to `ipc_radio` application and `hci_ipc` should not be used.

This PR migrates SUIT samples to the new `ipc_radio` sysbuild image.

Blocked by: NCSDK-28012

### CODE REVIEW

**Review by e-rk — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by robertstypa — APPROVED** [review]

File: samples/suit/recovery/sysbuild/ipc_radio.conf [hunk 1]
```diff
@@ -29,6 +25,7 @@ CONFIG_BT_ASSERT_VERBOSE=n
 CONFIG_ASSERT_VERBOSE=n
 CONFIG_ASSERT=n
 
+CONFIG_NCS_BOOT_BANNER=n
 CONFIG_BOOT_BANNER=n
 CONFIG_EARLY_CONSOLE=n
 CONFIG_PRINTK=n
```
Comment (dchat-nordic) [inline samples/suit/recovery/sysbuild/ipc_radio.conf:31]: Please add
```
CONFIG_IPC_RADIO_BT=y
CONFIG_IPC_RADIO_BT_HCI_IPC=y
```

File: samples/suit/recovery/sysbuild.conf [hunk 2]
```diff
@@ -6,7 +6,8 @@
 
 SB_CONFIG_SUIT_ENVELOPE=y
 
-SB_CONFIG_NETCORE_HCI_IPC=y
+SB_CONFIG_NETCORE_IPC_RADIO=y
+SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC=y
```
Comment (dchat-nordic) [inline samples/suit/recovery/sysbuild.conf:10]: Remove this line
```suggestion

```

File: samples/suit/recovery/sysbuild/ipc_radio.conf [hunk 3]
```diff
@@ -29,6 +25,7 @@ CONFIG_BT_ASSERT_VERBOSE=n
 CONFIG_ASSERT_VERBOSE=n
 CONFIG_ASSERT=n
```
Comment (dchat-nordic) [inline samples/suit/recovery/sysbuild/ipc_radio.conf:27]: Please add
```
CONFIG_MBOX=y
CONFIG_IPC_SERVICE=y

CONFIG_BT=y
CONFIG_BT_HCI_RAW=y
```

**Review by dchat-nordic — APPROVED** [review]
ipc_radio config looks good.
You need to update tests, because I see they look for hardcoded 'hci_ipc*' files.

**Review by FrancescoSer — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Pull request closed without merging**